[2022-09-13 13:17:28,390] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: first_py_dag.bash_dir 2022-09-13T13:17:24.472102+00:00 [queued]>
[2022-09-13 13:17:28,482] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: first_py_dag.bash_dir 2022-09-13T13:17:24.472102+00:00 [queued]>
[2022-09-13 13:17:28,484] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-13 13:17:28,497] {taskinstance.py:1018} INFO - Starting attempt 1 of 3
[2022-09-13 13:17:28,499] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-13 13:17:28,564] {taskinstance.py:1038} INFO - Executing <Task(BashOperator): bash_dir> on 2022-09-13T13:17:24.472102+00:00
[2022-09-13 13:17:28,585] {standard_task_runner.py:51} INFO - Started process 10033 to run task
[2022-09-13 13:17:28,697] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'first_py_dag', 'bash_dir', '2022-09-13T13:17:24.472102+00:00', '--job-id', '11', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_operator.py', '--cfg-path', '/tmp/tmpfqhslwv5']
[2022-09-13 13:17:28,743] {standard_task_runner.py:76} INFO - Job 11: Subtask bash_dir
[2022-09-13 13:17:29,096] {logging_mixin.py:103} INFO - Running <TaskInstance: first_py_dag.bash_dir 2022-09-13T13:17:24.472102+00:00 [running]> on host 27377494ab8b
[2022-09-13 13:17:29,273] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=first_py_dag
AIRFLOW_CTX_TASK_ID=bash_dir
AIRFLOW_CTX_EXECUTION_DATE=2022-09-13T13:17:24.472102+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-13T13:17:24.472102+00:00
[2022-09-13 13:17:29,276] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-09-13 13:17:29,281] {bash.py:158} INFO - Running command: ls
[2022-09-13 13:17:29,327] {bash.py:169} INFO - Output:
[2022-09-13 13:17:29,733] {bash.py:177} INFO - Command exited with return code 0
[2022-09-13 13:17:30,134] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=first_py_dag, task_id=bash_dir, execution_date=20220913T131724, start_date=20220913T131728, end_date=20220913T131730
[2022-09-13 13:17:30,950] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-09-13 13:17:31,034] {local_task_job.py:118} INFO - Task exited with return code 0
